### ISSUE TITLE
First child indented wrongly when multiple child with same name

### ISSUE THREAD

**Issue by wader** [clbanning/mxj#96]
Hi, here is a minimal reproduction case. Also maybe interesting to note that it seems to work fine if the "e" element level is removed.

https://go.dev/play/p/4iUMKbfcDL4
```go
package main

import (
	"encoding/json"
	"fmt"

	"github.com/clbanning/mxj/v2"
)

func main() {
	var v interface{}
	json.Unmarshal([]byte(`
{
  "doc": {
    "e": {
      "c": [
        "",
        ""
      ]
    }
  }
}
`), &v)
	m := mxj.Map(v.(map[string]interface{}))
	b, _ := m.XmlIndent("", "  ")
	fmt.Println(string(b))
}
```
Output:
```
<doc>
  <e>
      <c/>
    <c/>
  </e>
</doc>
```

**Comment by clbanning** [comment]
Will try to get to this in June.

**Comment by wader** [comment]
@clbanning 👍  No stress. I ended up using encoding/xml directly for some unrelated reasons, but got lot of inspiration from mxj!

**Comment by clbanning** [comment]
Fixed at tip, v2.5.6.

**Comment by wader** [comment]
👍
